Blog/view/tags.php

```diff
@@ -0,0 +1,48 @@
+<?php
+include('Blog/include/functions.php');
+$print = printHeader();
+ ?>
+	<head>
+		<!-- <title> Sort by Tags </title> -->
+		<link rel='stylesheet' href="/Blog/include/websiteMasterStyle.css">
+	</head>
+	<body>
+
+		<?php
+		 if ($_REQUEST['tagID'] == 0) {
+			echo "<title class='tagTitle'> Life Updates</title>";
```

Comment:
As a general rule, you should try to avoid situation where your code directly references primary keys in your db (e.g. a specific `tagID`).

Instead, take the tagID out of the request and call the database to get that tag's info. Then use that info to display the correct name in this title tag.